src/data/queries/healthServices.ts

```diff
@@ -0,0 +1,31 @@
+import { FieldHealthServicesArray } from '@/types/drupal/field_type'
+import {
+  HealthService as FormattedHealthService,
+  HealthServices as FormattedHealthServices,
+} from '@/types/formatted/healthServices'
+import { QueryFormatter } from 'next-drupal-query'
+
+export const formatter: QueryFormatter<
+  FieldHealthServicesArray,
+  FormattedHealthServices
+> = (entities: FieldHealthServicesArray): FormattedHealthServices => {
+  return entities.map((entity): FormattedHealthService => {
+    const serviceEntity = entity.entity.field_service_name_and_descripti
+
+    return {
+      name: serviceEntity.name,
+      vetCenterTypeOfCare: serviceEntity.field_vet_center_type_of_care || null,
```

Comment:
do all these need `|| null` ? if the field comes back from jsonapi as null they should get it anyway, right?